addon/utils/selection.js

```diff
@@ -149,14 +150,25 @@ export default {
   },
 
   /**
+   * Determines whether a method exists in this environment
+   *
```

Comment:
There is one method that did not follow the space after the description in the docBlock. I have fixed it so that all are consistent now.